website/pages/chapters/assessing_performance.js

```diff
@@ -193,22 +237,37 @@ export default function AssessingPerformance() {
           dealing with the <IM math={`(x, y)`} /> pairs. So then selecting the
           model that generalizes best, would then mean choosing the one with the
           lowest true error.
+          <MatthewComment>
+            I don't understand how these last two sentences are related to the
+            excpectation. It seems you've given the definition of the
+            expectation, and then said this looks like an expectation??
+          </MatthewComment>
         </p>
 
+        <h3> (Matthew:) Exiting theory land, back to practical ML</h3>
         <p>
           In many real-life circumstances, it turns out to not be possible to
-          compute this true error. You might not know the exact distributions of
-          the houses and their prices of all possible houses you could see in
-          the future! So without access to all future data, how can we actually
-          compute the true error?
-        </p>
-
-        <p>
-          A very common technique in machine learning suggests that if you
-          aren't able to exactly compute something, you can try to estimate it.
-          That's what we will do here. The basic idea is to hide part of our
-          dataset from our ML algorithm and use that hidden data as a proxy for
-          "all future data" after the predictor is trained.
+          compute this true error. You <strike>might not</strike> won't know the
```

Comment:
Hah. You gotta be careful and hedge a bit since there will always be some case where you can compute it and people will be pissy at you for saying something like "always" or "never" when that's not actually the case. 

I found it's easier to use vaguer words here to appease the sticklers.